.github/workflows/build-and-deploy.yml

```diff
@@ -3,6 +3,9 @@ name: check, build, deploy hpc-carpentry.org
 on:
   push:
     branches: main
+  pull_request:
+    branches:
+      - main
```

Comment:
This change should be removed before merging

Reply:
Confirmed this is working in https://github.com/hpc-carpentry/hpc-carpentry.github.io/runs/2391477702 so removing this now